### PR TITLE
Add GitHub issue templates

### PR DESCRIPTION
**Pull request by JeanPaiva**
This PR is to add custom issue templates for the repositories.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+labels: 'triage, type: bug'
+---
+
+### Description
+...
+
+### Steps to reproduce
+1. 
+
+#### Final behavior
```
Comment (tomusborne) [inline .github/ISSUE_TEMPLATE/bug.md:14]: Maybe this should be "Expected behavior" first?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+...
+
+#### Expected behavior
```
Comment (tomusborne) [inline .github/ISSUE_TEMPLATE/bug.md:17]: And this should maybe be "Actual behavior"?

**Comment by JeanPaiva** [comment]
Fixed, way better, thanks.

**Comment by tomusborne** [comment]
Looks good to me!

**Comment by alvarompg** [comment]
I like this!

**Pull request merged**